test/Scripts.Tests/package-release.zip.snapshot

```diff
@@ -329,6 +327,8 @@ Runtime/Sentry.xml
 Runtime/Sentry.xml.meta
 Runtime/SentryInitialization.cs
 Runtime/SentryInitialization.cs.meta
+Runtime/System.Runtime.CompilerServices.Unsafe.dll
+Runtime/System.Runtime.CompilerServices.Unsafe.dll.meta
 Samples~/unity-of-bugs/Scenes.meta
```

Comment:
Hmm, actually... Should we even ship these libs when they're not aliased?

Reply:
i dont know?

Reply:
perhaps @bitsandfoxes knows

Reply:
We should definitely ship with these since they are our dependencies. 

But we've got a bit of a cat chasing its own tail situation here:
1. We alias our dependencies so we don't conflict with other packages
2. We don't alias that specific DLL to make it work with 2022.1
3. If there is any other package that has this DLL as a dependency they will conflict
4. Our solution to resolving that conflict would be to alias the DLL but we can't because 1.

Reply:
happy to abandon this and wait for Unity to fix the issue. but in the short term i think potentially having a warning for one dependency is better than (removing alias and) having warnings for all.

Reply:
so to summarise, we're going with this solution for now. Anything against merging this @bitsandfoxes ?

Reply:
It's not just a warning, since we depend on version 5.0.0, and some packages that conflicted in the past brought in version 4.0.0.

We should test this change with some known packages and validate it actually works. Ideally add an integration test: Add unity's collection package to the smoke test empty project?